specs/solid-indexing/index.html

```diff
@@ -102,13 +356,16 @@ <h2>Indexes</h2>
     <section>
       <h3>General indexes</h3>
 
-      <p>An index is a RDF document [[RDF11-CONCEPTS]] of type <code>idx:Index</code> containing entries which point to
-        instances conforming to a particular shape. The shape that the index or an entry is targeting is expressed with the
-        <code>idx:hasShape</code> predicate.
-      </p>
+      <p>An index is a RDF document [[RDF11-CONCEPTS]] of type <code>idx:Index</code>. It contains entries of type <code>idf:IndexEntry</code>, each one 
+        being linked to a shape with the <code>idx:hasShape</code> predicate.</p>
+        
+      <p>An entry MUST refer to either an instance or to a sub index with the respective predicates <code>idx:hasTarget</code> and <code>idx:hasSubIndex</code>.</p>
```

Comment:
```suggestion
      <p>An entry MUST refer to either resources matching the shape  or to another index referring to such resources, with predicate <code>idx:hasTarget</code> or <code>idx:hasSubIndex</code> respectively.</p>
```

Reply:
or, if you don't want to allow multiple values for hasTarget/hasSubIndex.

```suggestion
      <p>An entry MUST refer to a resource matching the shape  or to another index referring to such resources, with predicate <code>idx:hasTarget</code> or <code>idx:hasSubIndex</code> respectively.</p>
```